kotlin/internal/js/impl.bzl

```diff
@@ -42,9 +47,11 @@ def kt_js_library_impl(ctx):
         _utils.derive_module_name(ctx),
     )
 
+    kotlinc_options = ctx.attr.kotlinc_opts[KotlincOptions] if ctx.attr.kotlinc_opts else toolchain.kotlinc_options
+
     args.add_all(
         "--kotlin_js_passthrough_flags",
-        [
+        kotlinc_options_to_flags(kotlinc_options) + [
             "-source-map",
             "-Xir-produce-klib-dir",
```

Comment:
Do you know if there will be any issues with duplicate flags being passed here. For example, `kotlinc_options_to_flags` could return `"-Xir-produce-klib-dir"` which is already being set here in impl.bzl.

Reply:
If I undestand the implementation of `kotlinc_options_to_flags` correctly, most of these aren't available as kotlinc options, meaning that can't happen. But `-no-sdtlib` could be conflicting if the options include `{"include_stdlibs": {"all","stdlib"}` (and duplicated if `"none"`). So probably worth handling more generally anyway.